src/main/java/net/sf/jabref/logic/layout/format/RTFChars.java

```diff
@@ -199,4 +199,100 @@ private StringInt getPart(String text, int i, boolean commandNestedInBraces) {
+            return "D";
+        if(240 == c || 273 == c)
+            return "d";
+        if((200 <= c && c <= 203) || 274 == c || 276 == c || 278 == c || 280 == c || 282 == c)
+            return "E";
+        if((232 <= c && c <= 235) || 275 == c || 277 == c || 279 == c || 281 == c || 283 == c)
+            return "e";
+        if((284 == c || 286 == c) || 288 == c || 290 == c || 330 == c)
+            return "G";
+        if(285 == c || 287 == c || 289 == c || 291 == c || 331 == c)
+            return "g";
+        if(292 == c || 294 == c)
+            return "H";
+        if(293 == c || 295 == c)
+            return "h";
+        if((204 <= c && c <= 207) || 296 == c || 298 == c || 300 == c || 302 == c || 304 == c)
+            return "I";
+        if((236 <= c && c <= 239) || 297 == c || 299 == c || 301 == c || 303 == c || 305 == c)
+            return "i";
+        if(308 == c)
+            return "J";
+        if(309 == c)
+            return "j";
+        if(310 == c)
+            return "K";
+        if(311 == c)
+            return "k";
+        if(313 == c || 315 == c || 319 == c || 312 == c)
+            return "L";
+        if(314 == c || 316 == c || 320 == c || 322 == c)
+            return "l";
+        if(209 == c || 323 == c || 325 == c || 327 == c)
+            return "N";
+        if(241 == c || 324 == c || 326 == c || 328 == c)
+            return "n";
+        if((210 <= c && c <=214) || c == 216 || 332 == c || 334 == c)
+            return "O";
+        if((242 <= c && c <= 248 && 247 != c) || 333 == c || 335 == c)
+            return "o";
+        if(340 == c || 342 == c || 344 == c)
+            return "R";
+        if(341 == c || 343 == c || 345 == c)
+            return "r";
+        if(346 == c || 348 == c || 350 == c || 352 == c)
+            return "S";
+        if(347 == c || 349 == c || 351 == c || 353 == c)
+            return "s";
+        if(354 == c || 356 == c || 358 == c)
+            return "T";
+        if(355 == c || 359 == c)
+            return "t";
+        if((217 <= c && c <= 220) || 360 == c || 362 == c || 364 == c || 366 == c || 370 == c)
+            return "U";
+        if((249 <= c && c <= 251) || 361 == c || 363 == c || 365 == c || 367 == c || 371 == c)
+            return "u";
+        if(372 == c)
+            return "W";
+        if(373 == c)
+            return "w";
+        if(374 == c || 376 == c || 221 == c)
+            return "Y";
+        if(375 == c || 255 == c)
+            return "y";
+        if(377 == c || 379 == c || 381 == c)
+            return "Z";
+        if(378 == c || 380 == c || 382 == c)
+            return "z";
+        if(198 == c)
+            return "AE";
+        if(230 == c)
+            return "ae";
+        if(338 == c)
+            return "OE";
+        if(339 == c)
+            return "oe";
+        if(222 == c)
+            return "TH";
+        if(223 == c)
+            return "ss";
+        if(161 ==c)
+            return "!";
+        return "?";
```

Comment:
I don't  like the multiple return statements and no brackets. Please use a variable which gets returned once in the end of the method and use `if else` with brackets.